File: Code_VB/C18_AppLogic/IncludeFile.vb
<script runat="server">

Private Sub WriteFile(ByVal strText As String)

    Using writer As New System.IO.StreamWriter(Server.MapPath("test.txt"), True)
        Dim str As String
        str = (DateTime.Now.ToString() & " ") + strText
        writer.WriteLine(str)
        writer.Close()
    End Using

End Sub

</script>

File: Code_CS/C18_AppLogic/IncludeFile.cs
<script runat="server">
    void WriteFile(string strText)
    {
        using (System.IO.StreamWriter writer =
            new System.IO.StreamWriter(@"C:\users\public\test.txt", true))
        {
            string str;
            str = DateTime.Now.ToString() + "  " + strText;
            writer.WriteLine(str);
            writer.Close();
        }
    }
</script>
